DEVELOPMENT.rst

```diff
@@ -534,7 +534,7 @@ This project uses environment variables for building the
   current build is intended for a wheel. On Windows, this will involve
   renaming ``bezier.dll`` to a unique name (to avoid name collision) and
   updating ``_speedup*.pyd`` to refer to the new name.
-- ``BEZIER_IGNORE_VERSION_CHECK ``: Will instruct ``pip`` and ``setup.py`` to
+- ``BEZIER_IGNORE_VERSION_CHECK``: Will instruct ``pip`` and ``setup.py`` to
```

Comment:
@behackl Sadly this will not pass CI because I have a template (`DEVELOPMENT.rst.template`) that I check this file against and a lint check to make sure they agree.

I am happy to merge with CI broken and fix the template file myself. But if you'd like to update the template file that's OK too.

----

A post-script on **why** there is a template. The template file has variables `revision` (i.e. `master` or `2021.2.12`) and `rtd_version` (i.e. `latest` or `stable` or `2021.2.12`). For most states of the mainline branch, these just refer to `master` and `latest` docs. However for **releases** (e.g. https://bezier.readthedocs.io/en/2021.2.12/development.html), I want these references to be frozen-in-time for that given release. So as part of a release, I use the template to update `DEVELOPMENT.rst` with the "frozen" `revision` and `rtd_version`.

Reply:
I see, that makes sense -- I've applied the same change to `DEVELOPMENT.rst.template` as well in 37124970. Should be good now.